src/app/freelancer/project/[project_id]/page.tsx

```diff
@@ -48,28 +53,64 @@ interface Project {
   updatedAt?: Date;
 }
 
+const getStatusBadge = (status: string | undefined) => {
+  switch (status?.toLowerCase()) {
+    case 'active':
+      return { text: 'ACTIVE', className: 'bg-blue-500 hover:bg-blue-600' };
```

Comment:
Are these status strings used anywhere else in project?
Please move these status strings (active, pending etc) to a object with key value pair in a constant file and use from there.